src/Microsoft.Health.Fhir.Api/Features/Audit/AuditHelper.cs

```diff
@@ -84,5 +92,29 @@ private void Log(AuditAction auditAction, HttpStatusCode? statusCode, HttpContex
                     customHeaders: _auditHeaderReader.Read(httpContext));
             }
         }
+
+        /// <summary>
+        /// Return all the values of constants of the specified type
+        /// </summary>
+        /// <returns>List of constant values</returns>
+        public static IList<string> GetConstants()
```

Comment:
nit: ambiguous name, perhaps `GetAnonymousOperations`.

The data type also looks like it could be `Lazy<List<string>>`

Reply:
Updated the PR as per suggestions